experimental/dds/tree2/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -142,6 +143,7 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 	protected summarizeCore(
 		serializer: IFluidSerializer,
 		telemetryContext?: ITelemetryContext,
+		incrementalSummaryContext?: IExperimentalIncrementalSummaryContext | undefined,
```

Comment:
```suggestion
		incrementalSummaryContext?: IExperimentalIncrementalSummaryContext,
```
(it's redundant with the `?`)

Reply:
removed undefined